examples/permissioned-token/scripts/4-setup-new-permissions-app.js

```diff
@@ -12,20 +12,24 @@ async function setupNewPermissionsApp(runtimeEnv, deployer) {
 	await fundAccount(deployer, owner);
 
 	// deploy new permissions smart contract
-	const newPermissionsAppInfo = await deployer.deployApp(
-		owner,
-		{
-			appName: "PermissionNewApp",
-			metaType: types.MetaType.FILE,
-			approvalProgramFilename: "permissions_new.teal", // new permissions contract
-			clearProgramFilename: "clear_state_program.py",
-			localInts: 0,
-			localBytes: 0,
-			globalInts: 0,
-			globalBytes: 0,
-		},
-		{}
-	);
+	const newPermissionsAppInfo = await deployer
```

Comment:
maybe we could create something similar for the deployApp as tryExecuteTx? I am not sure however since this will make it more disconnected from the way we should always use it

Reply:
I think you feel the issue more since we have multiple examples to handle, but for the user they might use these method once or twice, so it would make sense to have within the same file